mlflow/cli.py

```diff
@@ -94,18 +94,21 @@ def run(uri, entry_point, version, param_list, experiment_id, mode, cluster_spec
             sys.exit(1)
         param_dict[name] = value
     try:
-        projects.run(uri,
-                     entry_point,
-                     version,
-                     experiment_id=experiment_id,
-                     parameters=param_dict,
-                     mode=mode,
-                     cluster_spec=cluster_spec,
-                     git_username=git_username,
-                     git_password=git_password,
-                     use_conda=(not no_conda),
-                     use_temp_cwd=new_dir,
-                     storage_dir=storage_dir)
+        projects.run(
+            uri,
+            entry_point,
+            version,
+            experiment_id=experiment_id,
+            parameters=param_dict,
+            mode=mode,
+            cluster_spec=cluster_spec,
+            git_username=git_username,
+            git_password=git_password,
+            use_conda=(not no_conda),
+            use_temp_cwd=new_dir,
+            storage_dir=storage_dir,
+            block=True,  # TODO(sid) should users be able to launch async runs via the CLI?
```

Comment:
Maybe just add a note in the documentation that this will block until the run is completed; we can add such a parameter later but it should probably not be a TODO.

Reply:
I think it's nice to support async runs via the CLI too but we can do it in a later PR.